quiche/src/lib.rs

```diff
@@ -1897,6 +1897,11 @@ impl Connection {
             }
         }
 
+        // Reset pacer and start a new burst if packet received.
+        if done > 0 {
+            self.recovery.pacer.reset();
```

Comment:
This will be called even when an invalid packet is called (e.g. random Internet junk), is this ok? Maybe just move this at the end of `recv_single()`? This way we can also avoid the `Instant::now()` call inside, and just pass the `now` we already have.

Reply:
Good idea. Moved into recv_single().